js/explore/pie_repos.js

```diff
@@ -77,14 +77,14 @@ function draw_pie_repos(areaID) {
 
 		// Add legend
 		var legend = chart.selectAll('.legend')
-			.data(color.domain())
+			.data(data)
```

Comment:
I think this may have caused a bug where there is only one legend color in each.

Reply:
Caught that, now fixed.